packages/containers/src/actionAPI.js

```diff
@@ -16,14 +19,13 @@ function evalExpressions(action, context, payload = {}) {
 		context,
 		payload,
 	);
-	if (action.labelExpression) {
-		delete newAction.labelExpression;
-		newAction.label = api.expression.call(action.labelExpression, context, newAction);
-	}
-	if (action.iconExpression) {
-		delete newAction.iconExpression;
-		newAction.icon = api.expression.call(action.iconExpression, context, newAction);
-	}
+	forIn(action, (value, key) => {
```

Comment:
I prefer Object.keys().forEach which do not need lodash

Reply:
we already have loadash on our codebase so.